docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -425,11 +426,16 @@ To use these properties there are some specific conditions. For instance, you ca
 
 The following example illustrates how to use the `InputColor` property in a `TextField`:
 
+
+```xaml
```

Comment:
Seeking clarification on lines 430-438: It seems that the code is not consistent with the context above, can you please check this? 
Line 427 says the property "InputColor" has been used however, we couldn't find it in the code.

Reply:
updated